cmd/private-org-sync/main.go

```diff
@@ -460,35 +460,73 @@ func main() {
 		git:                  gitExec,
 		failOnNonexistentDst: o.failOnNonexistentDst,
 	}
+	var errs []error
+	var locations []location
+
+	whitelist := o.WhitelistOptions.WhitelistConfig.Whitelist
+	if len(whitelist) > 0 {
+		whitelistedLocations, err := getWhitelistedLocations(whitelist, syncer.git)
+		if err != nil {
+			errs = append(errs, err)
```

Comment:
Hmm, `err` is an aggregate anyway, so would it make sense to return a `[]error` instead from `getWhitelistedLocations` and simply append here? Would make the final aggregate more flat, which I think is better.